src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -30,7 +31,16 @@ We help you get to the heart of your system by offering two quick and easy ways
         src={nrqlconsole}
     />    
 
-You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages. 
+You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages.
+
+New Relic AI leverages a variety of skills and tools to complete analysis on your behalf, including:
+Natural language to query (NRQL, NerdGraph) translation
+Anomaly detection
+Error log analysis
+Error stack trace analysis
+Docs synthesis
```

Comment:
Might want to expand on this - not clear what this means and I don't think it's mentioned elsewhere in the doc